packages/splits-sdk/src/client/splitV2.ts

```diff
@@ -280,13 +273,19 @@ class SplitV2Transactions extends BaseTransactions {
         {
           recipients: recipientAddresses,
           allocations: recipientAllocations,
-          totalAllocation,
-          distributionIncentive,
+          totalAllocation: recipientAllocations.reduce(
+            (acc, curr) => acc + curr,
+            BigInt(0),
+          ),
+          distributionIncentive: getNumberFromPercent(
+            split.distributorFeePercent,
+          ),
         },
         token,
         distributorAddress,
       ],
       transactionOverrides,
+      value: ZERO,
```

Comment:
is this necessary? would have thought it's the default. fine to leave, just curious